File: tys-application/tys-commons/src/main/java/co/com/mintrabajo/tys/commons/domain/DatosEscenarioPractica.java
package co.com.mintrabajo.tys.commons.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

@Data
@AllArgsConstructor
@Builder(builderMethodName = "newInstance")
public class DatosEscenarioPractica {
    private String nombreEntidad;
    private Long idTipoEntidad;
    private Long idTipoIdentificacion;
    private String numeroIdentificacion;
    private Direccion direccion;
    private String correo;
    private String telefonoFijo;
    private String telefonoCelular;
}

File: tys-spa/ciudadano/src/app/domain/datosEscenarioPractica.ts
import {Direccion} from './direccion';

export interface DatosEscenarioPractica {
  nombreEntidad: string;
  idTipoEntidad: number;
  idTipoIdentificacion: number;
  numeroIdentificacion: string;
  direccion: Direccion;
  correo: string;
  telefonoFijo: string;
  telefonoCelular: string;
}
